### PR TITLE
Nsidc access

### PR DESCRIPTION
**Pull request by asteiker**

### CODE REVIEW

File: book/_toc.yml [hunk 1]
```diff
@@ -27,19 +27,20 @@
       sections:
         - file: tutorials/day_1
           sections:
-            - file: tutorials/jupyter
-            - file: tutorials/core-datasets/index
-              sections:
-                - file: tutorials/core-datasets/01_introduction
-                - file: tutorials/core-datasets/02_data-package
-                - file: tutorials/core-datasets/03_practice-querying
```
Comment (jomey) [inline book/_toc.yml:35]: @asteiker - This looks like a faulty merge and the yml the indentation needs to remain unchanged for day 1

**Review by jomey — CHANGES_REQUESTED** [review]

File: book/_toc.yml (hunk 1 above)
Comment (asteiker) [reply book/_toc.yml:35]: @jomey I _think_ I fixed the indents. Let me know if you still see anything amiss.

Comment (jomey) [reply book/_toc.yml:35]: Hm .. Sorry to be a pain here and yml can be challenging.

Child entries under a section should be followed with two spaces:
```yml
- file: A
  sections:
    - file B:
      sections:
```
Currently I see four under the section for day 1 and day 2

**Comment by jomey** [comment]
@asteiker - Learned something new and the preview for the TOC looks alright with the four spaces. Maybe you can change update the TOC to consistent spacing with your mentioned follow up PR to your tutorial.
Approving this for starters and feel free to merge.

**Review by jomey — APPROVED** [review]

**Pull request merged**